### PR TITLE
static method -> associated function

### PR DESCRIPTION
**Pull request by steveklabnik**

### CODE REVIEW

File: src/rust-installer [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit ebc6b04c29591108d3f28e724b4b9b74cd1232e6
+Subproject commit e54d4823d26cdb3f98e5a1b17e1c257cd329aa61
```
Comment (alexcrichton) [inline src/rust-installer]: Accidental change?

**Comment by alexcrichton** [comment]
r=me with a small nit, thanks!

**Comment by brson** [comment]
It looks to me like somebody reverted rust-installer in a previous patch.

File: src/rust-installer (hunk 1 above)
Comment (steveklabnik) [inline src/rust-installer]: yes, ugh

**Pull request merged**